python_modules/dagster-graphql/dagster_graphql/implementation/fetch_runs.py

```diff
@@ -128,18 +128,40 @@ def get_in_progress_runs_for_job(graphene_info, job_name):
 def get_in_progress_runs_by_in_progress_step(graphene_info, in_progress_runs, step_keys):
```

Comment:
nit: maybe we should rename this to `get_in_progress_runs_by_step`?

Reply:
good call